.gitlab-ci.yml

```diff
@@ -151,6 +199,70 @@ Integration:
         INTERNAL_NETWORK: ["true"]
         DISTRO_CODE: ["rhel_90"]
 
+# Disabled due to a missing commit in latest rpms
+# see COMPOSER-929 for more info
+#Integration-nightly-libvirt:
+#  stage: test
+#  extends: .terraform
+#  variables:
+#    EXTRA_REPO_PATH_SEGMENT: "gitlab/"
+#  script:
+#    - schutzbot/deploy.sh
+#    - /usr/libexec/tests/osbuild-composer/libvirt.sh
+#  parallel:
+#    matrix:
+#      - RUNNER:
+#          - openstack/rhel-8.4-x86_64
+#        INTERNAL_NETWORK: ["true"]
+#
+#Integration-nightly-aws:
+#  stage: test
+#  extends: .terraform
+#  rules:
+#    - if: '$CI_PIPELINE_SOURCE == "schedule"'
+#  variables:
+#    EXTRA_REPO_PATH_SEGMENT: "gitlab/"
+#  script:
+#    - schutzbot/deploy.sh
+#    - /usr/libexec/tests/osbuild-composer/aws.sh
+#  parallel:
+#    matrix:
+#      - RUNNER:
+#          - aws/rhel-8.4-x86_64
+#        INTERNAL_NETWORK: ["true"]
+
+Integration-koji-nightly:
+  stage: test
+  extends: .terraform
+  rules:
+    - if: '$CI_PIPELINE_SOURCE == "schedule"'
+  variables:
+    EXTRA_REPO_PATH_SEGMENT: "gitlab/"
+  script:
+    - schutzbot/deploy.sh
+    - /usr/libexec/tests/osbuild-composer/koji.sh
+  parallel:
+    matrix:
+      - RUNNER:
+          - aws/rhel-8.4-x86_64
+        INTERNAL_NETWORK: ["true"]
+
+Integration-api-nightly:
+  stage: test
+  extends: .terraform
+  rules:
+    - if: '$CI_PIPELINE_SOURCE == "schedule"'
+  variables:
+    EXTRA_REPO_PATH_SEGMENT: "gitlab/"
+  script:
+    - schutzbot/deploy.sh
+    - /usr/libexec/tests/osbuild-composer/api.sh
+  parallel:
+    matrix:
+      - RUNNER:
+          - aws/rhel-8.4-x86_64
+        INTERNAL_NETWORK: ["true"]
+
```

Comment:
hmmm, again, not liking the duplication and the possibility of testing to diverge over time and the necessity to maintain both job definitions. 

Is it possible to specify conditions per runner ? If that's possible we may say something like: 
- don't run Fedora/CentOS in schedules
- skip libvirt tests in schedules b/c of bugs, etc.
-

Reply:
I've extended from both Integration and Base stages and that seems to work well.